addons/t9n/models/message.py

```diff
@@ -11,8 +11,17 @@ class Message(models.Model):
     _description = "Localizable message"
 
     body = fields.Text(
+        "Entry to be Translated",
         help="The actual, textual content to be translated.",
     )
+    context = fields.Char(
+        help="Contect provided as `msgctxt` in the resource file."
+    )
+    translator_comments = fields.Text(
+        help="Comments wrote by the translator/developer in the resource file.")
+    extracted_comments = fields.Text(
+        help="Comments extracted during the process of extracting translatable strings."
+    )
```

Comment:
If you're going to split those over multiple lines, don't forget trailing commas